perllib/FixMyStreet/DB/Result/Comment.pm

```diff
@@ -290,6 +300,10 @@ sub meta_line {
 
     }
 
+    if ($self->get_extra_metadata('defect_raised')) {
+        $update_state .= ( $update_state ? ', ' : '' ) . _( 'and a defect raised' );
```

Comment:
I think putting this in `$update_state` will mean the next update, even if the state stays the same, will lead to a "marked as action scheduled" message because `$update_state` itself will have changed.